.travis/install.sh

```diff
@@ -1,17 +1,5 @@
 #!/bin/bash
 
-scripts_dir=$(dirname $(readlink -e "${BASH_SOURCE[0]}"))
-source "$scripts_dir/common"
+source "${0%/*}/setup.sh"
```

Comment:
nit: it's (in general) better to resolve symlink before going to parent dir. Here is the cross-platform way of doing it (thanks to Python and @dzhulgakov):
`script_path=$(python -c "import os; import sys; print(os.path.realpath(sys.argv[1]))" "${BASH_SOURCE[0]}")`

Reply:
++